src/logger.ts

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
this should be disabled project-wide until we get types for things everywhere, I thought?

Reply:
Well, I am explicitly using `any[]` for the args, so I don't see how the use of `any` can stop in this file.

This project has been newly converted to TS, so it doesn't have that linting rule setting at the project level because there hasn't been a need to yet.